packages/wallet/api/src/lib-wallet.js

```diff
@@ -1606,6 +1634,7 @@ export function makeWallet({
       return paymentsNotifier;
     },
     getUINotifier,
+    getPublicNotifiers,
```

Comment:
> We may want to deprecate getUINotifier at some point

Why not now? For reader of the code it helps explain why there are two methods.
```
    /** @deprecated use getPublicNotifiers instead */
    getUINotifier,
    getPublicNotifiers,
```

Reply:
I'm okay with that as long as @michaelfig agrees. I was on the fence of whether we'd also want to support the single notifier for simpler contracts, but it's probably better to just standardize.

Reply:
> I'm okay with that as long as @michaelfig agrees. I was on the fence of whether we'd also want to support the single notifier for simpler contracts, but it's probably better to just standardize.

I'm fine with deprecating, but not removing quite yet.

Reply:
Added `@deprecated` doc comment